.github/workflows/experimental.yml

```diff
@@ -2,6 +2,10 @@ name: 'Experimental Releases'
 
 on:
   workflow_dispatch:
+    inputs:
+      dist-tag:
+        description: 'The tag you want to remove from NPM'
```

Comment:
Should this be "The tag you want to publish to NPM" ?

Reply:
nice catch :)